### DISCUSSION TITLE
/MT vs /MD pre-built libraries

### DISCUSSION THREAD

**Discussion by ThibaultDECO** [jedisct1/libsodium#1215] [General]
It would be worth mentioning in the documentation ([see here](https://doc.libsodium.org/installation#pre-built-libraries)) that MSVC pre-built libraries are built with Multi-threaded (/MT) and not with Multi-threaded DLL (/MD). I'm a beginner in C++ and this got me into a bit of trouble.

**Comment by jedisct1** [comment]
I must confess that I don't know what the difference is.

Would you mind submitting a PR to explain this in the documentation? The documentation has its own repository: https://github.com/jedisct1/libsodium-doc

**Comment by markdascher** [comment]
@Teebonne I wonder if this is only true for the ltcg and static builds? I've linked against the pre-built dynamic DLL without any trouble, and my project is built with `/MD`. For example I currently link against `libsodium\x64\Release\v142\dynamic\libsodium.lib` and then copy the same folder's `libsodium.dll` next to my EXE before running it.

The ltcg and static libs seem to be built with `/MT` as you describe.
